concent_api/concent_api/system_check.py

```diff
@@ -539,3 +557,17 @@ def check_verifier_download_chunk_size(app_configs=None, **kwargs):  # pylint: d
             )]
 
     return []
+
+
+@register()
+def check_signing_service_key_availability_for_middleman(app_configs=None, **kwargs):  # pylint: disable=unused-argument
+    errors = []
+    if "middleman" in settings.CONCENT_FEATURES:
```

Comment:
Should `SIGNING_SERVICE_PUBLIC_KEY` be set when `middleman` is not in `CONCENT_FEATURES` ? Just asking, I remember that with verification use case we have some settings that were not supposed to be present without `verifier` or `conductor`.

Reply:
I think @cameel should answer that question, but he's on vacation right now. And there are no settings that can't be set, when we disable one of our apps, I've just checked.

Reply:
I'm not dead yet :)

It doesn't matter all that much.  It's fine either way.

Initially, for most settings I specified that they must be unset when we're not running an app that uses them. The goal was to easily spot errors in deployment such as setting something in the wrong app.

But it's a bit cumbersome so I'm on the fence on whether to continue doing that or not. I guess people stopped doing this when I stopped mentioning it in the issues so it's possible that we don't have too many settings like that any more.

Reply:
I'm leaving it as it is then.